modules/drupal-ce.yml

```
@@ -0,0 +1,17 @@
+name: drupal-ce
+description: Connects Nuxt.js with Drupal via the Lupus Custom Elements Renderer
+repo: drunomics/nuxt-module-drupal-ce
+npm: nuxtjs-drupal-ce
+icon: drupal-ce.png
+github: https://github.com/drunomics/nuxt-module-drupal-ce
```

Looks like the repo was renamed?
```suggestion
repo: drunomics/nuxtjs-drupal-ce
npm: nuxtjs-drupal-ce
icon: drupal-ce.png
github: https://github.com/drunomics/nuxtjs-drupal-ce
```

---

you are right - thanks for catching that! @davision just fixed it and updated maintainers section also.